### PR TITLE
Set lastest state of job groups to config

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: _review/qec-publiccloud.toml [hunk 1]
```diff
@@ -22,38 +22,35 @@ MaxLifetime = 86400
...
 Params = { flavor = "GCE-Updates", build = "%yesterday%-1" }
 [[Groups]]
 Name = "GCE-BYOS-Updates"
 Params = { flavor = "GCE-BYOS-Updates", build = "%yesterday%-1" }
+[[Groups]]
+Name = "GCE-CHOST-BYOS-Updates"
+Params = { flavor = "GCE-CHOST-BYOS-Updates", build = "%yesterday%-1" }
+[[Groups]]
+Name = "Server-DVD-Updates"
+Params = { flavor = "GCE-CHOST-BYOS-Updates", build = "%yesterday%-1", groupid = "427" }
```
Comment (grisu48) [inline _review/qec-publiccloud.toml]:
```suggestion
Name = "Azure CLI"
Params = { flavor = "Server-DVD-Updates", build = "%yesterday%-1", groupid = "427" }
```

Comment (asmorodskyi) [reply _review/qec-publiccloud.toml]: make sense , but it is not just Azure in theory it suppose to be ALL clients in a fact currently it is EC2 and GCE and Azure is skipped due to never-ending-bug

Comment (asmorodskyi) [reply _review/qec-publiccloud.toml]: WDYT about `Name = "Public Cloud clients"` ?

Comment (asmorodskyi) [reply _review/qec-publiccloud.toml]: sorry , due to the way git highlighted the diff I lost main point of your suggestion . this is fixed now but about actual group name I still think that `Azure CLI ` does not match

Comment (grisu48) [reply _review/qec-publiccloud.toml]:
```
Name = "Public Cloud clients"
```
Would be fine by me as well, just "GCE-CHOST-BYOS-Updates" was wrong. As long as the name is descriptive and points to the right thing, I don't mind.

File: _review/qec-publiccloud.toml [hunk 2]
```diff
@@ -72,6 +69,23 @@ Name = "EC2 Latest"
...
 
+# Staging 
+
+[[Groups]]
+Name = "Staging Azure"
+Params = { groupid = "463", build = "" }
+MaxLifetime = 86400
+
+[[Groups]]
+Name = "Staging - GCE"
```
Comment (grisu48) [inline _review/qec-publiccloud.toml:80]:
```suggestion
Name = "Staging GCE"
```
Consistent naming.

Comment (asmorodskyi) [reply _review/qec-publiccloud.toml:80]: done

**Review by grisu48 — APPROVED** [review]

**Comment by grisu48** [comment]
Thank you! :slightly_smiling_face:

**Pull request merged**